lib/models/cyclomatic_complexity_parameters.dart

```diff
@@ -0,0 +1,14 @@
+/// Complexity input parameters
```

Comment:
```suggestion
/// A data model class that represents the cyclomatic complexity input paramters.
```
Please, use [Effective Dart](https://dart.dev/effective-dart/documentation#prefer-starting-library-or-type-comments-with-noun-phrases) best practices for documenting your code. Ensure all other doc comments also follows it.